.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+exclude: (?x)(\.txt)
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.8.0
+    hooks:
+    -   id: black
+        args: [--safe, --quiet]
+-   repo: https://github.com/asottile/blacken-docs
+    rev: v1.12.1
+    hooks:
+    -   id: blacken-docs
+        additional_dependencies: [black==22.8.0]
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: fix-encoding-pragma
+        args: [--remove]
+    -   id: check-yaml
+        exclude: tests
+    -   id: check-toml
+    -   id: check-json
+    -   id: check-merge-conflict
+    -   id: pretty-format-json
+        args: [--autofix]
+    -   id: debug-statements
+        language_version: python3
+-   repo: https://github.com/pre-commit/mirrors-isort
```

Comment:
I strongly prefer https://github.com/asottile/reorder_python_imports, for the reasons cited there: https://github.com/asottile/reorder_python_imports#why-this-style

Having said that, I would then change the order things execute:

- autoflake
- reoder-imports
- black
- flake8

The rationale being: cleanups, formatters, flake checking

Reply:
I am not strongly attached to it, if you prefer this it is fine for me :)

Reply:
Sure lets do it then. Make sure to also remove `.isort.cfg` as it will no longer be needed.

Reply:
done!